mythril/laser/ethereum/keccak_function_manager.py

```diff
@@ -0,0 +1,106 @@
+from ethereum import utils
+from mythril.laser.smt import (
+    BitVec,
+    Function,
+    URem,
+    symbol_factory,
+    ULE,
+    And,
+    ULT,
+    Bool,
+    Or,
+)
+from typing import Dict, Tuple, List
+
+TOTAL_PARTS = 10 ** 40
+PART = (2 ** 256 - 1) // TOTAL_PARTS
+INTERVAL_DIFFERENCE = 10 ** 30
+hash_matcher = "fffffff"  # This is usually the prefix for the hash in the output
```

Comment:
All our hashes will have this as a prefix(assuming hash count isn't too high).Due to the way they are generated.

Reply:
fffffff is a fairly common string, and how high is too high? If it would be easy choose a more unique string, that might be better, but I don't think its critical at all. Feel free to resolve this issue if there's no easy improvement here.

Reply:
`how high is too high` pretty high, `>>> 10**10`(many orders higher than that).
Also I am computing inverse without model completion right? If it's a false positive then it's inverse will be just an unconstrained symbol which will result in skipping the case.

Reply:
Yeah, that looked right to me. The false positive detection seems fine.
 
The only potential false negatives that I'm seeing is if they're not aligned in memory, or if the input is shorter/longer than 64 bytes.

Reply:
yeah, that is related to the previous comment.

Reply:
`The only potential false negatives that I'm seeing is if they're not aligned in memory, or if the input is shorter/longer than 64 bytes.` This is being checked on the tx_data output right? so would it have any alignment problem now with these changes.

Reply:
It's not going to be very common, but it is feasible to come up with cases where it is possible for a keccak hash to not be aligned in the return data. Custom assembly will allow for this easily, and even in solidity, if bytes are returned and if they contain a hash, that hash could feasibly not be word-aligned.

Reply:
Now I am checking with every part of the word, that should do it right?